executor/executor_test.go

```diff
@@ -1882,3 +1882,25 @@ func (s *testSuite) TestFuncREPEAT(c *C) {
 	r = tk.MustQuery("SELECT REPEAT(a, 16777217), REPEAT(b, 16777217), REPEAT(c, 16777217), REPEAT(d, 16777217), REPEAT(e, 16777217), REPEAT(f, 16777217) FROM table_string;")
 	r.Check(testkit.Rows("<nil> <nil> <nil> <nil> <nil> <nil>"))
 }
+
+func (s *testSuite) TestEncyptionBuiltin(c *C) {
+	defer func() {
+		s.cleanEnv(c)
+		testleak.AfterTest(c)()
+	}()
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	// for sha2
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t(origin char(32), len int)")
+	tk.MustExec(`insert t values ("pingcap", 0)`)
+	tk.MustExec(`insert t values ("pingcap", 224)`)
+	tk.MustExec(`insert t values ("pingcap", 256)`)
+	tk.MustExec(`insert t values ("pingcap", 512)`)
```

Comment:
add test for hash length of 384
add test for hash length out of {224, 256, 384, 512, 0}